tests/UkModulusCheck.Tests/paket.references

```diff
@@ -0,0 +1,6 @@
+Expecto
+FSharp.Core
+Microsoft.DotNet.Watcher.Tools
```

Comment:
`dotnet watch` is built-in for new SDK, I believe.

Reply:
Yes, it is - it originates from MiniScaffold which still uses 2.0 SDK, if we're sure we can use 2.1 then I'll remove it.